recipe/meta.yaml

```diff
@@ -23,30 +22,34 @@ build:
 
 requirements:
   host:
```

Comment:
```suggestion
  build:
    - {{ compiler('c') }}
  host:
```
Because anaconda-linter complains:
`cython_needs_compiler: Cython generates C code, which will need to be compiled`
It's required if you enable `FONTTOOLS_WITH_CYTHON=1`

Reply:
Added the compiler!